tests/cypress/page-objects/BasePage.js

```diff
@@ -0,0 +1,38 @@
+export class BasePage {
+  visit(pageUrl) {
+    cy.visit(pageUrl);
+  }
+
+  pageReload() {
+    cy.reload();
+  }
+
+  resizeMedium() {
+    cy.viewport(1024, 640);
+  }
+
+  resizeXXsmall() {
+    cy.viewport(480, 640);
+  }
+
+  resizeXLarge() {
+    cy.viewport(1400, 840);
```

Comment:
Do these viewports correspond with specific breakpoints? If so, can you leave a comment indicating why we chose these sizes in this file?